examples/long-short.py

```diff
@@ -11,328 +12,275 @@
+                if self.short.count(position.symbol) == 0:
+                    # Position not in short list either.  Clear position.
+                    if position.side == "long":
+                        side = "sell"
+                    else:
+                        side = "buy"
+                    self.submit_order(abs(int(float(position.qty))), position.symbol, side)
+                else:
+                    # Position in short list.
+                    if position.side == "long":
+                        # Position changed from long to short.  Clear long position to prepare for short position.
+                        side = "sell"
+                        self.submit_order(int(float(position.qty)), position.symbol, side)
+                    else:
+                        if abs(int(float(position.qty))) == self.q_short:
+                            # Position is where we want it.  Pass for now.
+                            pass
+                        else:
+                            # Need to adjust position amount
+                            diff = abs(int(float(position.qty))) - self.q_short
+                            if diff > 0:
+                                # Too many short positions.  Buy some back to rebalance.
+                                side = "buy"
+                            else:
+                                # Too little short positions.  Sell some more.
+                                side = "sell"
+                            self.submit_order(abs(diff), position.symbol, side)
+                        executed[1].append(position.symbol)
+                        self.blacklist.add(position.symbol)
+            else:
+                # Position in long list.
+                if position.side == "short":
+                    # Position changed from short to long.  Clear short position to prepare for long position.
+                    self.submit_order(abs(int(float(position.qty))), position.symbol, "buy")
+                else:
+                    if int(float(position.qty)) == self.q_long:
+                        # Position is where we want it.  Pass for now.
+                        pass
+                    else:
+                        # Need to adjust position amount.
+                        diff = abs(int(float(position.qty))) - self.q_long
+                        if diff > 0:
+                            # Too many long positions.  Sell some to rebalance.
+                            side = "sell"
+                        else:
+                            # Too little long positions.  Buy some more.
+                            side = "buy"
+                        self.submit_order(abs(diff), position.symbol, side)
+                    executed[0].append(position.symbol)
+                    self.blacklist.add(position.symbol)
+
+        # Send orders to all remaining stocks in the long and short list.
+        resp_send_bo_long = self.send_batch_order(self.q_long, self.long, "buy")
```

Comment:
[minor] tuple unpacking would be much nicer:
```suggestion
        executed, incomplete = self.send_batch_order(self.q_long, self.long, "buy")
```